Sources/MultiSlider.swift

```diff
@@ -198,10 +200,10 @@ open class MultiSlider: UIControl {
         var delta = snapStepSize > 0 ? stepSizeInView : thumbViews[draggedThumbIndex].frame.size(in: orientation) / 2
         if orientation == .horizontal { delta = -delta }
         let bottomLimit = draggedThumbIndex > 0
-            ? thumbViews[draggedThumbIndex - 1].center.coordinate(in: orientation) - delta
+            ? thumbViews[draggedThumbIndex - 1].center.coordinate(in: orientation) - (keepsDistanceBetweenThumbs ? delta : 0)
```

Comment:
I think it would be cleaner to change `delta` (line 200). Then you’d only check this condition once, and not need to have it duplicated in two places.

Reply:
Done :)